sos/sosreport.py

```diff
@@ -567,6 +567,8 @@ def load_plugins(self):
         self.profiles = set()
         using_profiles = len(self.opts.profiles)
         policy_classes = self.policy.valid_subclasses
+        # load kmods for plugin enablement checks
+        self.policy.lsmod()
```

Comment:
Do we want to be driving this from `sosreport`, or should it be auto-initialised either during policy initialisation or on first use (i.e. have a `@property` method that initialises it)?

It seems a bit out of keeping with the other system data we make available through policies to have this explicitly initialised by the user (e.g. `PackageManager` and `InitSystem`).